Packs/Twitter/ReleaseNotes/2_0_0.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"Twitter v1 is deprecated, Use Twitter v2 instead."}
```

Comment:
Add here the difference in the commands from v1 to v2
What has changes, what is deprecated and not used